### ISSUE TITLE
python 3.7.1 "ValueError: Cannot set the NumPy array 'base' dependency more than once"

### ISSUE THREAD

**Issue by notmyrealhandle** [scikit-sparse/scikit-sparse#49]
5 of 10 nosetests fail with the error "ValueError: Cannot set the NumPy array 'base' dependency more than once" on python 3.7.1 but all 10 nosetests pass on python 3.7.0 (details about my installation below).

From a jupyter qt console:

**Comment by notmyrealhandle** [comment]
Thank you for the quick response.

Reverting to Cython 0.28.5 works for me and is something I can live with for now.

As this seems like an upstream problem, do you want me to close this?

**Comment by anntzer** [comment]
Let's fix that here too.

**Comment by mcepl** [comment]
I don't think it is a Cython problem: I can (un)happily reproduce it with Cython 0.29 as well, and that should have https://github.com/cython/cython/pull/2528 merged in.

**Comment by anntzer** [comment]
I'm not saying it's a Cython bug; rather, the abovementioned Cython PR exposed a problem in the implementation of sksparse and I'm proposing to workaround it by pinning to an old version of Cython (at least for now).

**Comment by mcepl** [comment]
@anntzer Sure, but it is not a solution. When I am building packages for OpenSUSE, I cannot hold some package (especially as important as Cython) back.

Just for the completeness sake, here is the [complete build log](https://github.com/scikit-sparse/scikit-sparse/files/2548508/_log.txt) from an attempt to build scikit-sparse on openSUSE.

**Comment by anntzer** [comment]
You are welcome to PR a fix.
In the meantime you could build the package from within a venv, with the suitable version of cython installed in it.

**Comment by jor-** [comment]
I'm looking into the problem right now. And hope to publish a fix soon that also works with the current NumPy and Cython version.

**Comment by mcepl** [comment]
Could we get release on PyPI?

**Comment by jor-** [comment]
Yes, I am going to make one today.

**Comment by jor-** [comment]
A PyPI release is now also available.
